libpages/root.go

```diff
@@ -68,11 +68,26 @@ type Root struct {
 	PathUnparsed    string
 }
 
-// MakeFS makes a *libfs.FS from *r, which can be adapted to a http.FileSystem
+// CacheableFS is a wrapper around a *libfs.FS and a subdir. Use use() to get a
+// *libfs.FS that roots at subdir. This essentially delays "cd"ing into subdir,
+// and is useful for caching a *libfs.FS object without the downside of caching
+// a libkbfs.Node that can be obsolete when it's renamed or removed.
+type CacheableFS struct {
```

Comment:
Does this need to be exported?

Reply:
Yeah because `MakeFS` is exported and golint would complain otherwise.

Reply:
Cool thx